locales/bn/json.json

```diff
@@ -1076,24 +1076,24 @@
+    "You are currently within your free trial period. Your trial will expire on :date. Starting a new subscription will end your trial.": "আপনি বর্তমানে আপনার বিনামূল্যে ট্রায়াল সময়ের মধ্যে আছে. আপনার ট্রায়ালের মেয়াদ :date. তারিখে শেষ হবে একটি নতুন সাবস্ক্রিপশন শুরু করলে আপনার ট্রায়াল শেষ হবে।",
     "You are logged in!": "আপনি লগ ইন আছেন!",
     "You are not eligible for this plan.": "আপনি এই পরিকল্পনার জন্য যোগ্য নন।",
     "You are receiving this email because we received a password reset request for your account.": "আমরা আপনার অ্যাকাউন্টের জন্য একটি পাসওয়ার্ড রিসেট অনুরোধ পেয়েছি, কারণ আপনি এই ইমেইল পাচ্ছেন ।",
-    "You currently have :count invitation(s) remaining.": "আপনার বর্তমানে :countটি আমন্ত্রণ বাকি আছে।",
+    "You currently have :count invitation(s) remaining.": "আপনার বর্তমানে :count-টি আমন্ত্রণ বাকি আছে।",
     "You currently receive a discount of :discountAmount for a single invoice.": "আপনি বর্তমানে একটি একক চালানের জন্য :discountAmount ছাড় পাবেন।",
-    "You currently receive a discount of :discountAmount for all invoices during the next :months billing cycles.": "আপনি বর্তমানে পরবর্তী :monthsটি বিলিং চক্র চলাকালীন সমস্ত চালানের জন্য :discountAmountটি ছাড় পাচ্ছেন৷",
+    "You currently receive a discount of :discountAmount for all invoices during the next :months billing cycles.": "আপনি বর্তমানে পরবর্তী :months-টি বিলিং চক্র চলাকালীন সমস্ত চালানের জন্য :discountAmountটি ছাড় পাচ্ছেন৷",
     "You currently receive a discount of :discountAmount for all invoices during the next :months months.": "আপনি বর্তমানে পরবর্তী :months মাসে সমস্ত চালানের জন্য :discountAmount ছাড় পাবেন।",
     "You currently receive a discount of :discountAmount for all invoices during the next billing cycle.": "আপনি বর্তমানে পরবর্তী বিলিং চক্রের সময় সমস্ত চালানের জন্য :discountAmount ছাড় পাবেন।",
     "You currently receive a discount of :discountAmount for all invoices during the next month.": "আপনি বর্তমানে পরের মাসে সমস্ত চালানের জন্য :discountAmount ডিসকাউন্ট পাবেন।",
     "You currently receive a discount of :discountAmount forever.": "আপনি বর্তমানে চিরতরে :discountAmount ছাড় পাবেন।",
     "You have been invited to join the :team team!": "আপনি :team দলের যোগদানের জন্য আমন্ত্রণ জানানো হয়েছে!",
-    "You have cancelled your subscription to the :planName plan.": "আপনি :planName প্ল্যানে আপনার সদস্যতা বাতিল করেছেন।",
-    "You have enabled two factor authentication.": "আপনি দুটি ফ্যাক্টর প্রমাণীকরণ সক্রিয় আছে ।",
-    "You have not enabled two factor authentication.": "আপনি দুটি ফ্যাক্টর প্রমাণীকরণ সক্রিয় করা হয়নি ।",
+    "You have cancelled your subscription to the :planName plan.": "আপনি :planName প্ল্যানে আপনার সাবস্ক্রিপশন বাতিল করেছেন।",
+    "You have enabled two factor authentication.": "আপনি দুটি ফ্যাক্টর প্রমাণীকরণ সক্রিয় আছে।",
+    "You have not enabled two factor authentication.": "আপনি দুটি ফ্যাক্টর প্রমাণীকরণ সক্রিয় করা হয়নি।",
```

Comment:
The sentence "আপনি দুটি ফ্যাক্টর প্রমাণীকরণ সক্রিয় করা হয়নি।" should be "আপনি দুটি ফ্যাক্টর প্রমাণীকরণ সক্রিয় করেননি।"